src/main/java/com/nina/modelling/shoes/KidShoe.java

```diff
@@ -0,0 +1,34 @@
+package com.nina.modelling.shoes;
+
+public class KidShoe extends Shoe {
```

Comment:
Please answer for each of the trheads with what was changed, but dont mark as resolved.
I will mark resolve if I agree with them else i will reopen for fix

Reply:
Ok